### PR TITLE
feature/add-svelte-ui

### PR DESCRIPTION
**Pull request by silvestrevivo**
#### feature/add-svelte-ui

This PR creates a Svelte version of the React UI package using the exact same style.css stylesheet.

It exposes the following components:

- `WalletMultiButton` as entry-point of the ui implementation
- `WalletButton` as wrapper button for the different variants
- `WalletConnectButton` connects the selected wallet.
- `WalletModal` modal component containing the wallets to be selected

#### Usage:

Insert the `<WalletMultiButton` in your Svelte application and that's all. The UI implementation pull the methods and store from the [core](https://github.com/solana-labs/wallet-adapter/pull/161)

To complete the integration and following what is done in the React and Vue implementation, the next components are added:

- `<WalletProvider />`
- `<ConnectionProvider />`

To give support to this, we need to add this Svelte Store. 

- `workSpace.ts`

This last one file, is a Svelte Store where are saved the values from Connection that should be used/share among the components to make/sign transactions.

To show how this works, a [repo](https://github.com/silvestrevivo/solana-svelte-counter) is created as [demo](https://solana-svelte-counter.netlify.app/) which is deployed in Netlify connected to the DevNet.

#### Package

Followed the SvelteKit [packaging documentation](https://kit.svelte.dev/docs#packaging)

```
`npm run build` to package the app using SvelteKit
```

The output files will be in the `package` directory

### CODE REVIEW

**Comment by silvestrevivo** [comment]
@jordansexton Hi there!

This PR related to the UI is ready too.

**Comment by cogoo** [comment]
We tested this package locally with `npm link`

File: packages/ui/svelte-ui/src/lib/ConnectionProvider.svelte [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	import { Connection } from '@solana/web3.js';
+	import type { Commitment, ConnectionConfig } from '@solana/web3.js';
+	import { workSpace } from '@solana/wallet-adapter-svelte';
```
Comment (jpshankle) [inline packages/ui/svelte-ui/src/lib/ConnectionProvider.svelte:4]: It looks like workSpace comes from svelte-ui package not core svelte?

**Comment by cogoo** [comment]
@jpshankle thanks for looking at this. This PR is blocked until https://github.com/solana-labs/wallet-adapter/pull/173 is merged. 

We'll have to make some more amends based on the outcome of that PR

**Comment by cogoo** [comment]
@jordansexton and @jpshankle this is now ready to review.

Looking for some advice on how to handle https://github.com/solana-labs/wallet-adapter/pull/162/commits/477b385275728e901ecff35d735efcb31bbfbe1a

This UI package was tested in conjunction with the `wallet-adapter-svelte` package locally using symlinks.

This works both in SPA and SSR mode

**Comment by lenchik-ponchik** [comment]
Any progress on this?

**Comment by revick** [comment]
Sveltekit/Svelte dev numbers are skyrocketing; can the Svelte NPM packages also be pushed up the list.

**Comment by jordansexton** [comment]
#285 has removed Angular from this monorepo to its own repo where it can be maintained. I think we should do the same with Svelte and Vue in order to improve the release process and cadence for packages that I can't effectively maintain as part of the core. This will also allow these external packages to have more idiomatic patterns, build systems, and documentation.

@cogoo we discussed this before, but do want to take the lead on this?

**Comment by cogoo** [comment]
> #285 has removed Angular from this monorepo to its own repo where it can be maintained. I think we should do the same with Svelte and Vue in order to improve the release process and cadence for packages that I can't effectively maintain as part of the core. This will also allow these external packages to have more idiomatic patterns, build systems, and documentation.
> 
> @cogoo we discussed this before, but do want to take the lead on this?

Work on this has already started in https://github.com/svelte-on-solana. Few things left to iron out before first alpha release. Soon™

**Comment by jordansexton** [comment]
Awesome! Whenever you think it's ready, please open a PR to remove it and add links to your repo's packages.

I'm excited to finally unblock Svelte users :)

**Comment by jordansexton** [comment]
Svelte support is going to become community maintained! Please subscribe to https://github.com/solana-labs/wallet-adapter/pull/298, which will be completed and merged when @cogoo and @silvestrevivo release Svelte packages.

**Pull request closed without merging**